docker-compose.override.example.yml

```diff
@@ -144,3 +144,37 @@ services:
   zookeeper:
     ports:
       - ${ZOOKEEPER_PUBLISH_PORT}:2181
+
+  coordinator:
+    volumes:
+      - common-druid-shared:/opt/shared
+      - common-coordinator-var:/opt/druid/var
+    ports:
+      - ${APACHE_DRUID_COORDINATOR_PUBLISH_PORT}:8081
+
+  broker:
+    volumes:
+      - common-broker-var:/opt/druid/var
+    ports:
+      - ${APACHE_DRUID_BROKER_PUBLISH_PORT}:8082
+
+  historical:
+    volumes:
+      - common-druid-shared:/opt/shared
+      - common-historical-var:/opt/druid/var
```

Comment:
Move to `docker-compose.override.apache-druid.yml` file

Reply:
Volumes sections for Apache Druid is moved from `docker-compose.override.example.yml` to `docker-compose.override.apache-druid.yml` file.